tokio-macros/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "tokio-macros"
 # - Create "tokio-macros-1.x.y" git tag.
 version = "2.1.0"
 edition = "2021"
-rust-version = "1.56"
+rust-version = "1.63"
```

Comment:
tokio-macros is a dependency of tokio, so I don't think it is a utility crate.

Reply:
Good point. I'll reword the commit message when I merge it. However, I don't see any issue in upgrading it, since the next release of tokio-macros will need at least the next version of Tokio, which is on 1.63.